lib/view_model/exchange/exchange_view_model.dart

```diff
@@ -40,8 +40,9 @@ class ExchangeViewModel = ExchangeViewModelBase with _$ExchangeViewModel;
 abstract class ExchangeViewModelBase with Store {
   ExchangeViewModelBase(this.wallet, this.trades, this._exchangeTemplateStore,
       this.tradesStore, this._settingsStore, this.sharedPreferences) {
-    const excludeDepositCurrencies = <CryptoCurrency>[];
-    const excludeReceiveCurrencies = [CryptoCurrency.xlm, CryptoCurrency.xrp, CryptoCurrency.bnb];
+    const excludeDepositCurrencies = [CryptoCurrency.xhv, CryptoCurrency.btt, CryptoCurrency.nano];
```

Comment:
Please remove XHV from excluded deposit currencies, as it was added to support in another branch.